extensions/notebook/package.json

```diff
@@ -329,6 +329,15 @@
           "Notebook"
         ],
         "configuration": "./language-configuration.json"
+      }, {
+        "id": "dib",
```

Comment:
Adding this language definition here for now to get things working. We can discuss adding this to the .NET Interactive extension directly at a later date. Interestingly, VS Code doesn't require a language contribution like this to get new notebook file types to save properly (i.e. .dib shows up first in list of valid file types). It seems like they do some magic underneath by inferring file type from the notebook format's registered file extensions.